pages/itinerary.js

```diff
@@ -41,6 +67,13 @@ export default function Itinerary({ trips, sights, setSights }) {
 
   const datesArray = createItinerary();
 
+  function formatTime(sight) {
+    const [hours, minutes] = sight.plannedTime.split(":");
+    const mode = parseInt(hours) >= 12 ? "pm" : "am";
+    const formattedTime = `${hours % 12}.${minutes} ${mode}`;
+    return formattedTime;
```

Comment:
Minor improvement here. We can directly return the formattedTime rather than creating a variable for it.  
```suggestion
    return `${hours % 12}.${minutes} ${mode}`;
```